privacy/bolt_on/losses.py

```diff
@@ -237,9 +235,9 @@ def __init__(self,
       dtype: tf datatype to use for tensor conversions.
     """
     if label_smoothing != 0:
-      logging.warning("The impact of label smoothing on privacy is unknown. "
-                      "Use label smoothing at your own risk as it may not "
-                      "guarantee privacy.")
+      print("WARNING: The impact of label smoothing on privacy is unknown. "
```

Comment:
Does it work to simply use python logging module? If not, we can also use absl-py library according to this page: https://www.tensorflow.org/beta/guide/effective_tf2

Reply:
switched back to absl.logging, was not working originally